opm/parser/eclipse/EclipseState/SimulationConfig/tests/ThresholdPressureTest.cpp

```diff
@@ -181,13 +181,10 @@ static DeckPtr createDeck(const ParseContext& parseContext , const std::string&
     return parser.parseString(input , parseContext);
 }
 
-static Eclipse3DProperties getEclipseProperties(const Deck& deck)
-{
-    static const EclipseGrid eclipseGrid( 3, 3, 3 );
-    return Eclipse3DProperties( deck, TableManager( deck ), eclipseGrid );
+static Eclipse3DProperties getEclipseProperties(const Deck& deck) {
+    return Eclipse3DProperties(deck, *new TableManager(deck), *new EclipseGrid(3, 3, 3));
```

Comment:
Now the link (in the email) to this comment failed to work due to rebasing. Just saying...

I think this is unacceptable even if its "just a test". Also, tests are used as guidelines and examples for people who want to use a feature, and we run the risk of teaching very bad habits.